x-pack/plugins/reporting_export_types/kibana.jsonc

```
@@ -0,0 +1,17 @@
+{
+  "type": "plugin",
+  "id": "@kbn/reporting-export-types",
+  "owner": "@elastic/appex-sharedux",
+  "plugin": {
+    "id": "reportingExportTypes",
+    "server": true,
+    "browser": false,
+    "configPath": [
+      "xpack",
+      "reportingExportTypes"
+    ],
```

Looks like this causes the server to crash, since there are no plugins that export a config schema with this path. 

For now, we can re-use the configPath from `reporting`:
```suggestion
    "configPath": [
      "xpack",
      "reporting"
    ],
```